### ISSUE TITLE
False positive bad-python3-import for stat module

### ISSUE THREAD

**Issue by adiroiban** [PyCQA/pylint#1565]
### Steps to reproduce

I have this test file

```
from __future__ import absolute_import
import stat

stat.ingore
```

### Current behavior

```
pylint --py3k test.py 
************* Module test
W:  2, 0: Module moved in Python 3 (bad-python3-import)

------------------------------------------------------------
An exception occurred while rating: name 'no' is not defined
```

### Expected behavior

`import stat` should not raise an issue

I am not a py3 migration expert, but at least on my system i get

```
$ python
Python 2.7.12 (default, Nov 19 2016, 06:48:10) 
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import stat
>>> 

$ python3
Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import stat
>>> 
```

### pylint --version output

```
pylint --version
pylint 1.7.1, 
astroid 1.5.3
Python 2.7.13 (4f815b5, May  2 2017, 11:54:59) 
[GCC 5.4.0 20160609]
```
